packages/tree/src/Treemap.ts

```diff
@@ -41,6 +42,21 @@ export class Treemap extends HTMLWidget {
         Utility.SimpleSelectionMixin.call(this);
     }
 
+    getTilingMethod() {
+        switch (this.tilingMethod()) {
+             case "treemapBinary":
```

Comment:
Make this the "default" also (just in case the use manages to specify an invalid tilingMEthod?

Reply:
I added the default at the end (and changed the default to "treemapSquarify")